src/renderer/components/node/IteratorNode.tsx

```diff
@@ -31,17 +32,14 @@ export const IteratorNode = memo(({ data, selected }: IteratorNodeProps) => (
 
 const IteratorNodeInner = memo(({ data, selected }: IteratorNodeProps) => {
     const { schemata, categories } = useContext(BackendContext);
+    const { useIteratorProgress } = useContext(ExecutionContext);
+
+    const { id, inputData, isLocked, schemaId, iteratorSize, minWidth, minHeight } = data;
+
+    const { getIteratorProgress } = useIteratorProgress(id);
+
+    const { percent, eta } = getIteratorProgress();
```

Comment:
Why expose `useIteratorProgress` when only `getIteratorProgress` is ever used outside `ExecutionContext`?

Reply:
When i was writing this i thought i needed these and i just didn't remove them